src/components/Sponsors/Sponsors.js

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+import React from "react";
+import "./Sponsors.css";
 
 function Sponsors() {
   return (
-    <div>Hi Maritza!</div>
+    <div class="sponsors-container">
+      <div class="sponsors-title">
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">P</div>
+          <div class="letter">O</div>
+          <div class="letter">N</div>
+        </div>
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">O</div>
+          <div class="letter">R</div>
+          <div class="letter">S</div>
+        </div>
+      </div>
+
+      <div class="sponsors-row">
+        <div class="box">
+          We are still looking for sponsors for GraceHacks 2020. Please reach
+          out to us if you're interested in helping make this hackathon a
+          success.
+        </div>
+      </div>
+
+      <div class="sponsors-row">
+        <div class="button">SPONSOR GRACE HACKS TODAY!</div>
```

Comment:
Run this through Saniya and ask her if we're still accepting sponsors. If not, we can remove this.

Reply:
I checked with her and she says we still are, so I left it for now